package.json

```diff
@@ -16,30 +16,28 @@
     "url": "https://github.com/raintank/app-plugin-worldping/issues"
   },
   "devDependencies": {
-    "babel": "~6.5.1",
-    "grunt": "~0.4.5",
+    "babel-cli": "^6.24.1",
```

Comment:
Moved dependencies to "devDependency" section, as they are not required to run the plugin once it's built.